CHANGELOG.asciidoc

```diff
@@ -56,6 +55,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - When collecting swap metrics for beats telemetry or system metricbeat module handle cases of free swap being bigger than total swap by assuming no swap is being used. {issue}6271[6271] {pull}9383[9383]
 - Adding logging traces at debug level when the pipeline client receives the following events: onFilteredOut, onDroppedOnPublish. {pull}9016[9016]
 - Ignore non index fields in default_field for Elasticsearch. {pull}9549[9549]
+- Update to Golang 1.11.3. {pull}9560[9560]
```

Comment:
I think the first one becomes obsolete?

Reply:
Well, a changelog is a trace of what was done so I am not sure I would remove that.

Reply:
I would argue this changelog here is for the users and it only matters what happened between releases. If we introduce a bug and fix it before it is released, we also don't list it here.

Reply:
I've made the change.